smartling-job-batches-api/src/test/java/com/smartling/api/jobbatches/v2/JobBatchesApiTest.java

```diff
@@ -81,7 +82,7 @@ public void testCreateBatch() throws Exception
             + "\"translationJobUid\":\"jobUid\","
             + "\"authorize\":true,"
             + "\"fileUris\":[\"fileUri.json\"],"
-            + "\"localeWorkflows\":[{\"targetLocaleId\":\"fr-FR\",\"workflowUid\":\"workflowUid\"}]"
+            + "\"localeWorkflows\":[{\"targetLocaleId\":\"fr-FR\",\"workflowUid\":\"workflowUid\",\"contentAssignments\":null}]"
```

Comment:
"contentAssignments" field is not required and nullable.

So I think it will be ok to use it without contentAssignments as now.

Tests were added in another PR to cover this case: https://github.com/Smartling/jobs-facade/pull/135

Reply:
agree with pavel, I'd remain this test data as is, and add another with contentAssignements field.

Reply:
Tests were added in another PR to cover this case: https://github.com/Smartling/jobs-facade/pull/135

Reply:
<img width="1009" alt="image" src="https://github.com/user-attachments/assets/be85e57c-b7ee-4783-8a0e-fa4e842ed4eb">

Here I setuped input pto without contentAssignments array, as is on prod for now.
requestString contains contentAssignments = null because request.getBody().readUtf8() prints all fields from pto's with default values if such field explicitly is not defined.